diff --git a/base/BUILD.gn b/base/BUILD.gn
index 34e650277f6ea5a80e99f2e556f29c408585540a..d3729450de63148a6ce2222355e4d61c39356340 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4033,6 +4033,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/TaskTraits.java",
       "android/java/src/org/chromium/base/task/TaskTraitsExtensionDescriptor.java",
     ]
+    sources += brave_java_base_sources
 
     annotation_processor_deps = [ "//base/android/jni_generator:jni_processor" ]
 
