diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index f1a6bf1cccdabb6ad961f0f14c8199279e574010..f18fc96ae99329862daa452d8eb2a225daa6a454 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -344,6 +344,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
