diff --git a/base/BUILD.gn b/base/BUILD.gn
index a8684cc2bf48a7df896847ad1b8f34308db3c28f..ffbfc06ddb4057e5d05595e92d4afd4284c069cc 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4495,6 +4495,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/ValueChangedCallback.java",
       "android/java/src/org/chromium/base/WrappedClassLoader.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -5027,6 +5028,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_library("flagged_apis_java") {
