diff --git a/base/test/BUILD.gn b/base/test/BUILD.gn
index 2f894643ff8b2d49200c2a15a38138153fa55705..b79c1758cba4e83a63a0d8df0277b2aacc7e2f3d 100644
--- a/base/test/BUILD.gn
+++ b/base/test/BUILD.gn
@@ -344,6 +344,7 @@ static_library("test_support") {
       "trace_to_file.h",
     ]
   }
+  import("//brave/base/sources.gni") sources += brave_base_test_support_sources
 }
 
 component("proto_test_support") {
