diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index f061c661e40138bbd3086cf8a04b0ea20bd00403..a264dd4ad887756e725f8b9dbb07460bf3f013ae 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1662,6 +1662,7 @@ config("chromium_code") {
       ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [
     ":default_warnings",
