diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index 67b7097dd9450dc9d38048312907dfcffef2c179..f82727af3fda8d2976eccedfbfb1a25ce4b4b1e9 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1634,6 +1634,7 @@ config("chromium_code") {
       cflags += [ "-Wextra-semi" ]
     }
   }
+  cflags -= brave_removed_chromium_code_cflags
 
   configs = [ ":default_warnings" ]
 }
