diff --git a/build/android/gyp/proguard.py b/build/android/gyp/proguard.py
index 3a12370a60ca5c1ac470cbe0bbc917e6fa4deb12..a326c3158b8f3af12b1c5794d6bc5915cf4b23b7 100755
--- a/build/android/gyp/proguard.py
+++ b/build/android/gyp/proguard.py
@@ -362,6 +362,7 @@ def _OptimizeWithR8(options, config_paths, libraries, dynamic_config_data):
         '--pg-map-output',
         tmp_mapping_path,
     ]
+    cmd.remove('--map-id-template'); cmd.remove(f'{options.source_file} ({options.package_name})') # required to omit package check for mapping file
 
     if options.uses_split:
       cmd += ['--isolated-splits']
