diff --git a/build/dotfile_settings.gni b/build/dotfile_settings.gni
index 10c0e325114b85160fd736dfbb5bd0088662fbfc..3239ba01360565311a4dd26dde95d847f43728f4 100644
--- a/build/dotfile_settings.gni
+++ b/build/dotfile_settings.gni
@@ -46,3 +46,4 @@ build_dotfile_settings = {
   # so that we don't need to keep supporting `exec_script_whitelist`.
   exec_script_whitelist = exec_script_allowlist
 }
+import("//brave/build/dotfile_settings.gni") build_dotfile_settings.exec_script_allowlist += brave_exec_script_allowlist
