diff --git a/build/config/BUILDCONFIG.gn b/build/config/BUILDCONFIG.gn
index b5c5cb75e5fe2c58b1ee9051a9fe8cc082b88efa..671c5c95934182b45ddb335b13340ab2b61c84b9 100644
--- a/build/config/BUILDCONFIG.gn
+++ b/build/config/BUILDCONFIG.gn
@@ -343,6 +343,7 @@ is_posix = !is_win && !is_fuchsia
 
 # Holds all configs used for running the compiler.
 default_compiler_configs = [
+  "//brave/build:compiler",
   "//build/config:feature_flags",
   "//build/config/compiler:afdo",
   "//build/config/compiler:afdo_optimize_size",
