diff --git a/chrome/browser/sync/sync_service_factory.cc b/chrome/browser/sync/sync_service_factory.cc
index 77da00b4285c8ce8d7c6e4ee8adf9d3dec4f5f39..e656760efd531b0cc63dab5579bc4e97312c3737 100644
--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -171,7 +171,7 @@ std::unique_ptr<KeyedService> BuildSyncService(
           init_params.sync_client.get());
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize(
       client_ptr->CreateDataTypeControllers(sync_service.get()));
 
