diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index d1aafd563e9e978a28c1c7a3b16d22a9af4afd76..ff8f69fddb4ae432b3ff933cc150e2e6358ce167 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -332,6 +333,7 @@ function createRoutes(): SettingsRoutes {
           loadTimeData.getString('performancePageTitle'));
     }
   }
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
