diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index aba92a8ce84224c73dd522d0977be2c42dea3725..4fbc2ee7e6ea20aa8edaa8040f1ba5c247b6f2e4 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3923,6 +3923,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
