diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index f5fa439b0b8c71e3a331a4235d673a263ae3c219..0795e590b42b1b8dfa0149e631fbd0e42a5d8521 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4018,6 +4018,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
