diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index c4bde64baa0879952c1bf75f5a728d05be6761f6..d4ba99c6d53282ecbcc0216d5be29ad3cbe98a5b 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1572,6 +1572,7 @@ source_set("extensions") {
       "api/tabs/app_window_helper.h",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions_core) {
