diff --git a/chrome/browser/ui/views/toolbar/toolbar_view.cc b/chrome/browser/ui/views/toolbar/toolbar_view.cc
index 9bfef2eceef767e2167535472f2d04482f43c9d2..2f3702d61ba6ecd652b379b38cfbea5006e242ab 100644
--- a/chrome/browser/ui/views/toolbar/toolbar_view.cc
+++ b/chrome/browser/ui/views/toolbar/toolbar_view.cc
@@ -453,6 +453,7 @@ void ToolbarView::Init() {
     }
   }
 
+  BRAVE_TOOLBAR_VIEW_INIT
   avatar_ = container_view_->AddChildView(
       std::make_unique<AvatarToolbarButton>(browser_view_));
   bool show_avatar_toolbar_button = true;
