diff --git a/chrome/browser/extensions/extension_tab_util.cc b/chrome/browser/extensions/extension_tab_util.cc
index 19549095b129403e62e34b32be2124a8a58d597e..af2b82e8baeefccf96b42651e4f89a365ad472cd 100644
--- a/chrome/browser/extensions/extension_tab_util.cc
+++ b/chrome/browser/extensions/extension_tab_util.cc
@@ -902,6 +902,7 @@ bool ExtensionTabUtil::IsKillURL(const GURL& url) {
   }
 
   if (!url.SchemeIs(content::kChromeUIScheme)) {
+    if (!url.SchemeIs(content::kBraveUIScheme))
     return false;
   }
 
