diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 958c3a3b3c2b077911aa8cb4c50f515a02a8b741..4a576728d4524354212df18a50882b44b05a9574 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4012,6 +4012,7 @@ const FeatureEntry::FeatureVariation kWebAuthnAndroidCredManVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
