diff --git a/chrome/browser/metrics/chrome_metrics_services_manager_client.cc b/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
index 9f8870262c83014339899b0e3b336cd5be4114e8..1fb1093823027294b0a2253e856fbd322eddc493 100644
--- a/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
+++ b/chrome/browser/metrics/chrome_metrics_services_manager_client.cc
@@ -154,7 +154,7 @@ class ChromeMetricsServicesManagerClient::ChromeEnabledStateProvider
   }
 
   bool IsReportingEnabled() const override {
-    return metrics::EnabledStateProvider::IsReportingEnabled() &&
+    return false && metrics::EnabledStateProvider::IsReportingEnabled() &&
            IsClientInSampleImpl(local_state_);
   }
 
