diff --git a/chrome/browser/extensions/component_loader.h b/chrome/browser/extensions/component_loader.h
index 5089843bbc37e89b09e400f864c2a0d60cf3198a..a39b4dacc8c00bdbe98c0844290899098e593a64 100644
--- a/chrome/browser/extensions/component_loader.h
+++ b/chrome/browser/extensions/component_loader.h
@@ -130,6 +130,7 @@ class ComponentLoader {
     ignore_allowlist_for_testing_ = value;
   }
 
+  BRAVE_COMPONENT_LOADER_H
  private:
   FRIEND_TEST_ALL_PREFIXES(ComponentLoaderTest, ParseManifest);
 
