diff --git a/chrome/browser/resources/side_panel/read_anything/BUILD.gn b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
index a02f11dcc0192a38c7b08f3622faa62f0b301480..c94f6c2a5106a9530548b4ade397602a500cd2fa 100644
--- a/chrome/browser/resources/side_panel/read_anything/BUILD.gn
+++ b/chrome/browser/resources/side_panel/read_anything/BUILD.gn
@@ -53,7 +53,7 @@ build_webui("build") {
             target_gen_dir) ]
 
   webui_context_type = "untrusted"
-  optimize = optimize_webui
+  optimize = false
   if (optimize) {
     optimize_webui_host =
         "chrome-untrusted://read-anything-side-panel.top-chrome"
