diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 55cc881cb6e86d05c3b5c9532cd3eb531a516481..bf197b38593cf541528323fe1d3808688496dbca 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -489,7 +489,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
   source->AddBoolean("isLacrosEnabled",
                      crosapi::browser_util::IsLacrosEnabled());
 #endif
-
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
