diff --git a/chrome/browser/profiles/profile_manager.h b/chrome/browser/profiles/profile_manager.h
index 8ba4e0ba9f3c014fb94794d50afe8d54e94e3fdf..9e0fd8ccd3e5d2f5bf78c3b66ff5ef89f318d5a7 100644
--- a/chrome/browser/profiles/profile_manager.h
+++ b/chrome/browser/profiles/profile_manager.h
@@ -349,6 +349,7 @@ class ProfileManager : public Profile::Delegate {
   void set_do_final_services_init(bool do_final_services_init) {
     do_final_services_init_ = do_final_services_init;
   }
+  BRAVE_PROFILE_MANAGER_H
 
 #if BUILDFLAG(IS_CHROMEOS_LACROS)
   void SetAccountProfileMapperForTests(
