diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 58b3b7c39d23aba29f45f39825b43e6f536cb460..d1fdfc5d8edfae3467ccfdebe1de340137d92d8a 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4609,6 +4609,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
