diff --git a/chrome/renderer/BUILD.gn b/chrome/renderer/BUILD.gn
index d6a0aa3766a9bb28b585954a1f3bb21c765a56fe..2951f4f0d72fe41ce079aa115d07b92c5e79008d 100644
--- a/chrome/renderer/BUILD.gn
+++ b/chrome/renderer/BUILD.gn
@@ -234,6 +234,7 @@ static_library("renderer") {
     "//v8",
   ]
 
+  import("//brave/renderer/sources.gni") sources += brave_chrome_renderer_sources deps += brave_chrome_renderer_deps public_deps += brave_chrome_renderer_public_deps
   data_deps = [ "//tools/v8_context_snapshot" ]
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
