diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index 1265719e1b83f08e79827e871e13e15883aea508..c6325944935e4f7d11b3fff0342e0e3bda472df6 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -490,6 +490,7 @@ class ExtensionService : public ExtensionServiceInterface,
   // While disabled all calls to CheckForExternalUpdates() will bail out.
   static base::AutoReset<bool> DisableExternalUpdatesForTesting();
 
+  BRAVE_EXTENSION_SERVICE_H
  private:
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
