diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 86dc7d83ec9b3ec390527fe0edb42c886348efd0..73d8055eb2fe5c9c053d5f0936bc3bf7977d59f8 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2300,6 +2300,7 @@ static_library("browser") {
     "//ui/web_dialogs",
     "//ui/webui",
     "//ui/webui/resources/cr_components/customize_themes:mojom",
+    "//brave/browser:browser_process",
   ]
   if (is_chromeos) {
     deps += [
@@ -4063,6 +4064,7 @@ static_library("browser") {
         sources += [ "first_run/first_run_internal_posix.cc" ]
       }
     }
+    sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps
   }
 
   if (is_chromeos) {
