diff --git a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
index f7d6c28a31774e03775bdb30c8ece773950fa6dc..bce3c5d0080678f91b0c4aee14c6871b7fedd39e 100644
--- a/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
+++ b/chrome/browser/ui/android/appmenu/internal/java/src/org/chromium/chrome/browser/ui/appmenu/AppMenu.java
@@ -332,6 +332,7 @@ class AppMenu implements OnItemClickListener, OnKeyListener, AppMenuClickHandler
                 anchorView,
                 groupDividerResourceId,
                 anchorViewOffset);
+        if (this instanceof BraveAppMenu) ((BraveAppMenu) this).updatePopup(mPopup, isByPermanentButton);
         int[] popupPosition =
                 getPopupPosition(
                         mTempLocation,
