diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 9af39526ebb1a2bdfd3b88e8dfa9b27cd15dbbbc..df7c36535c223596dd73589b66f70a2bc16ea1be 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4104,6 +4104,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
