diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 139ade66fbaf20f7c41275f5f84acf81afcc03f7..a0acc7a53fd1c0f1ae6c1054f631660414516158 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2623,6 +2623,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
