diff --git a/chrome/android/java/src/org/chromium/chrome/browser/compositor/scene_layer/TabListSceneLayer.java b/chrome/android/java/src/org/chromium/chrome/browser/compositor/scene_layer/TabListSceneLayer.java
index 654e433afa295a71d557bcd7a4d5276c73486919..89deea849a297fbd2dd74f37d5b1d0aade38e173 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/compositor/scene_layer/TabListSceneLayer.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/compositor/scene_layer/TabListSceneLayer.java
@@ -29,11 +29,11 @@ import org.chromium.ui.util.ColorUtils;
  */
 @JNINamespace("android")
 public class TabListSceneLayer extends SceneLayer {
-    private long mNativePtr;
+    public long mNativePtr;
     private TabModelSelector mTabModelSelector;
     private int[] mAdditionalIds = new int[4];
     private boolean mUseAdditionalIds;
-    private boolean mIsInitialized;
+    public boolean mIsInitialized;
 
     public void setTabModelSelector(TabModelSelector tabModelSelector) {
         mTabModelSelector = tabModelSelector;
