diff --git a/chrome/browser/extensions/extension_management.h b/chrome/browser/extensions/extension_management.h
index 1c4a68369bb5030e763a52b35573e7319d7fa641..b0c19b664a2a3c685e8e7c263a4d7d730a879d2d 100644
--- a/chrome/browser/extensions/extension_management.h
+++ b/chrome/browser/extensions/extension_management.h
@@ -229,6 +229,7 @@ class ExtensionManagement : public KeyedService {
   // Returns whether the profile associated with this instance is supervised.
   bool is_child() const { return is_child_; }
 
+  BRAVE_EXTENSION_MANAGEMENT_H
  private:
   using SettingsIdMap =
       std::unordered_map<ExtensionId,
