diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
index 05bf49c1d9046a180a4ca582f2f0e6811f805292..d4b1bf63c5996503d1c23aeb9abe124c87e780c8 100644
--- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
+++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
@@ -130,6 +130,7 @@ content::WebUIDataSource* CreateAndAddBookmarksUIHTMLSource(Profile* profile) {
   for (const auto& str : kStrings)
     AddLocalizedString(source, str.name, str.id);
 
+  BRAVE_CREATE_BOOKMARKS_UI_HTML_SOURCE
   return source;
 }
 
