diff --git a/chrome/browser/ui/tabs/tab_style.cc b/chrome/browser/ui/tabs/tab_style.cc
index 84fcc3fd98b67662e1f3bc6911f6f0516bfd0b88..3f3a8d98a431350ff55fa069e2d5d1a461b15499 100644
--- a/chrome/browser/ui/tabs/tab_style.cc
+++ b/chrome/browser/ui/tabs/tab_style.cc
@@ -195,6 +195,7 @@ float ChromeRefresh2023TabStyle::GetSelectedTabOpacity() const {
 
 // static
 const TabStyle* TabStyle::Get() {
+  BRAVE_TAB_STYLE_GET
   static TabStyle* const tab_style =
       static_cast<TabStyle*>(new ChromeRefresh2023TabStyle());
 
