diff --git a/chrome/browser/ssl/https_upgrades_interceptor.cc b/chrome/browser/ssl/https_upgrades_interceptor.cc
index cca12cf20c0fe0c0655159aaf2450227f8e54721..dee66ae25c8709fee1fcdc9fca1e41ca8784782b 100644
--- a/chrome/browser/ssl/https_upgrades_interceptor.cc
+++ b/chrome/browser/ssl/https_upgrades_interceptor.cc
@@ -238,6 +238,7 @@ bool ShouldExcludeNavigationFromUpgrades(
     level = NavigationRequestSecurityLevel::kCaptivePortalLogin;
   }
 #endif
+  BRAVE_SHOULD_EXCLUDE_NAVIGATION
 
   RecordNavigationRequestSecurityLevel(level);
   return true;
