diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 4178eaf9bd0318368e82a6b8f898db95da63dd5b..f821dc2bf9f22bbae66589ebd12498316989fedf 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2686,6 +2686,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
