diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index 1c0bbb956cde9d70c4fc4458a3e1153f3d453767..dff8d68aa261cb18bed6a13493ccd9242dd1076b 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -600,6 +600,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
     // Block any navigation requests in locked fullscreen mode.
     return nullptr;
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/1096345): Remove this code after we integrate with intent
@@ -967,6 +968,7 @@ bool IsHostAllowedInIncognito(const GURL& url) {
   if (scheme != content::kChromeUIScheme) {
     return true;
   }
+  BRAVE_IS_HOST_ALLOWED_IN_INCOGNITO
 
   if (host == chrome::kChromeUIChromeSigninHost) {
 #if BUILDFLAG(IS_WIN)
