diff --git a/chrome/browser/ui/tabs/tab_strip_model.cc b/chrome/browser/ui/tabs/tab_strip_model.cc
index 501eb5506077d47b5490d51c131ba5bee2748fd3..abab0dc9f48386590320088dd4d96391e1cc67f4 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.cc
+++ b/chrome/browser/ui/tabs/tab_strip_model.cc
@@ -1240,6 +1240,7 @@ void TabStripModel::RemoveFromGroup(const std::vector<int>& indices) {
 
 bool TabStripModel::IsReadLaterSupportedForAny(
     const std::vector<int>& indices) {
+  BRAVE_TAB_STRIP_MODEL_DEACTIVATE_READING_LIST
   if (profile_->IsGuestSession())
     return false;
 
