diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarButtonInProductHelpController.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarButtonInProductHelpController.java
index 94649324cf1b9250f015b5c8db99d6552cdcd828..97d345a4a7dab0d82be0cb6d151b0d662a5f36bb 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarButtonInProductHelpController.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarButtonInProductHelpController.java
@@ -36,6 +36,7 @@ import org.chromium.components.feature_engagement.FeatureConstants;
 import org.chromium.components.feature_engagement.Tracker;
 import org.chromium.content_public.browser.UiThreadTaskTraits;
 
+import org.chromium.chrome.browser.toolbar.bottom.BottomToolbarConfiguration;
 /**
  * A helper class for IPH shown on the toolbar.
  * TODO(https://crbug.com/865801): Remove feature-specific IPH from here.
@@ -190,7 +191,8 @@ public class ToolbarButtonInProductHelpController
 
     // Private methods.
     private static int getDataReductionMenuItemHighlight() {
-        return R.id.app_menu_footer;
+        return BottomToolbarConfiguration.isBottomToolbarEnabled() ? R.id.data_reduction_menu_item
+                                                                   : R.id.app_menu_footer;
     }
 
     // Attempts to show an IPH text bubble for data saver detail.
