diff --git a/chrome/browser/ui/webui/extensions/extensions_ui.cc b/chrome/browser/ui/webui/extensions/extensions_ui.cc
index 5d58ebb7ec8af5cd07ccc5350bad98d5fac656e7..55b2b86467f527f0c28d90d036d167f4a71b3afb 100644
--- a/chrome/browser/ui/webui/extensions/extensions_ui.cc
+++ b/chrome/browser/ui/webui/extensions/extensions_ui.cc
@@ -494,7 +494,7 @@ content::WebUIDataSource* CreateAndAddExtensionsSource(Profile* profile,
   source->AddBoolean("isLacrosEnabled",
                      crosapi::browser_util::IsLacrosEnabled());
 #endif
-
+  BRAVE_CREATE_EXTENSIONS_SOURCE
   return source;
 }
 
