diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 7bd876cedcd2852fc131a6e2f7a10e1a79254339..83b3149fcb6623cc0adf854667baa5844914dc6d 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2763,6 +2763,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
