diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
index a443e779bf34f168dd5fa5961dd95a14e2536065..ee9ceda6d54fdaf4e4b3564748bc4ed1f39a01ee 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
@@ -354,7 +354,7 @@ public class ToolbarManager
     }
 
     /** An {@link ObservableSupplier<Integer>} for the browser constraints of the current tab. */
-    private static class ConstraintsProxy extends ObservableSupplierImpl<Integer>
+    public static class ConstraintsProxy extends ObservableSupplierImpl<Integer>
             implements Callback<Integer> {
         private ObservableSupplier<Integer> mCurrentConstraintDelegate;
 
