diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 7f9e3dbcb82de525836510cda59f88cd667288a9..2a7db131a6e7674cfbb6958e0174e33d7891a7a1 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1467,6 +1467,7 @@ source_set("extensions") {
       allow_circular_includes_from += [ "//chrome/browser/glic:impl" ]
     }
   }  # if (enable_extensions)
+  deps += brave_extensions_resources
 }
 
 if (enable_extensions_core) {
