diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 823cad1fe7aa1be961c4fbb151adc67c282dafd4..aa1ef496a8bc9b25631f04fc45d4236c97181a05 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -116,6 +116,7 @@ source_set("configs") {
       }
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
