diff --git a/chrome/browser/ui/webui/BUILD.gn b/chrome/browser/ui/webui/BUILD.gn
index 1bc4ec41402ca4ca6d21232d798a75fa7eddf5cd..e4c42656f683f64f526d0afeb8b9f328cc05b248 100644
--- a/chrome/browser/ui/webui/BUILD.gn
+++ b/chrome/browser/ui/webui/BUILD.gn
@@ -142,6 +142,7 @@ source_set("configs") {
       }
     }
   }
+  import("//brave/browser/ui/webui/sources.gni") deps += brave_browser_ui_webui_configs_deps
 }
 
 source_set("webui") {
