diff --git a/chrome/common/BUILD.gn b/chrome/common/BUILD.gn
index 4cea6cd1d4e8a1cf5e8ff0f967dd33fec6900abc..831ec2d765c2f0f39c4d1c802cc38b644a5290a7 100644
--- a/chrome/common/BUILD.gn
+++ b/chrome/common/BUILD.gn
@@ -84,6 +84,7 @@ source_set("channel_info") {
   } else if (is_fuchsia) {
     sources += [ "channel_info_fuchsia.cc" ]
   }
+  public_deps += [ "//brave/common:channel_info" ]
 }
 
 source_set("ini_parser") {
@@ -550,6 +551,7 @@ static_library("non_code_constants") {
     "//printing/buildflags",
     "//ui/base:buildflags",
   ]
+  configs += [ "//brave/common:constants_configs" ]
 }
 
 component("chrome_features") {
