diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index aa54bb8c29e2b0bd3c811cbe0620a79bd7894505..496bb59a3a6e88b66dea9aa2e320061aac870b9b 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -7624,6 +7624,7 @@ const FeatureEntry kFeatureEntries[] = {
     // "LoginCustomFlags" in tools/metrics/histograms/enums.xml. See "Flag
     // Histograms" in tools/metrics/histograms/README.md (run the
     // AboutFlagsHistogramTest unit test to verify this process).
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 };
 
 class FlagsStateSingleton : public flags_ui::FlagsState::Delegate {
