diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 95c288ea917801626e445d9e4379d31287d1d980..ef33da4e97a70d0002290d4481f7f504b67407ff 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2719,6 +2719,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
