diff --git a/chrome/browser/android/compositor/scene_layer/tab_list_scene_layer.h b/chrome/browser/android/compositor/scene_layer/tab_list_scene_layer.h
index f998ad33112d9d9e47dc3d99b724969865c80b3a..3d8f3d5c25de28cac802987a1ccdd3643d416fe0 100644
--- a/chrome/browser/android/compositor/scene_layer/tab_list_scene_layer.h
+++ b/chrome/browser/android/compositor/scene_layer/tab_list_scene_layer.h
@@ -109,7 +109,7 @@ class TabListSceneLayer : public SceneLayer {
   bool ShouldShowBackground() override;
   SkColor GetBackgroundColor() override;
 
- private:
+ protected:
   typedef std::map<int, scoped_refptr<TabLayer>> TabMap;
   TabMap tab_map_;
   std::set<int> visible_tabs_this_frame_;
