diff --git a/chrome/browser/browser_process_impl.h b/chrome/browser/browser_process_impl.h
index c9efc2ac813add645ec7c806a8d0576640465425..bc7fe1af2a9e249098c698a4edcb276939cad702 100644
--- a/chrome/browser/browser_process_impl.h
+++ b/chrome/browser/browser_process_impl.h
@@ -242,6 +242,7 @@ class BrowserProcessImpl : public BrowserProcess,
   GlobalFeatures* GetFeatures() override;
 
  private:
+  friend class BraveBrowserProcessImpl;
   using WebRtcEventLogManager = webrtc_event_logging::WebRtcEventLogManager;
 
   // KeepAliveStateObserver implementation
