diff --git a/chrome/browser/metrics/variations/BUILD.gn b/chrome/browser/metrics/variations/BUILD.gn
index ccb8ef058361495a7dc1ded63e178d6a7a9f3cb8..c7b48d3e611f59b1bbf8041a2929dd4228fcd105 100644
--- a/chrome/browser/metrics/variations/BUILD.gn
+++ b/chrome/browser/metrics/variations/BUILD.gn
@@ -8,17 +8,17 @@ generate_ui_string_overrider("chrome_ui_string_overrider_factory") {
   inputs = [
     "$root_gen_dir/chrome/grit/chromium_strings.h",
     "$root_gen_dir/chrome/grit/generated_resources.h",
-    "$root_gen_dir/chrome/grit/google_chrome_strings.h",
+    #"$root_gen_dir/chrome/grit/google_chrome_strings.h",
     "$root_gen_dir/components/strings/grit/components_chromium_strings.h",
-    "$root_gen_dir/components/strings/grit/components_google_chrome_strings.h",
+    # "$root_gen_dir/components/strings/grit/components_google_chrome_strings.h",
     "$root_gen_dir/components/strings/grit/components_strings.h",
   ]
   deps = [
     "//chrome/app:chromium_strings",
     "//chrome/app:generated_resources",
-    "//chrome/app:google_chrome_strings",
+    #"//chrome/app:google_chrome_strings",
     "//components/strings:components_chromium_strings",
-    "//components/strings:components_google_chrome_strings",
+    # "//components/strings:components_google_chrome_strings",
     "//components/strings:components_strings",
   ]
   namespace = "chrome_variations"
