diff --git a/chrome/android/java/src/org/chromium/chrome/browser/night_mode/settings/ThemeSettingsFragment.java b/chrome/android/java/src/org/chromium/chrome/browser/night_mode/settings/ThemeSettingsFragment.java
index 02a30988975722a372af4fef4259e668a69ecf32..91290f99b8858c61a98de60711d2b50ed8ef1db4 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/night_mode/settings/ThemeSettingsFragment.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/night_mode/settings/ThemeSettingsFragment.java
@@ -23,7 +23,7 @@ import org.chromium.ui.UiUtils;
 /**
  * Fragment to manage the theme user settings.
  */
-public class ThemeSettingsFragment extends PreferenceFragmentCompat {
+public class ThemeSettingsFragment extends org.chromium.components.browser_ui.settings.BravePreferenceFragment {
     static final String PREF_UI_THEME_PREF = "ui_theme_pref";
 
     @Override
