diff --git a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
index 61f72eca877ab26214186806160be82b6f7525fb..c65ca0f06166ba63685038474384dcc122a7df57 100644
--- a/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
+++ b/chrome/browser/ui/toolbar/recent_tabs_sub_menu_model.h
@@ -82,6 +82,7 @@ class RecentTabsSubMenuModel : public ui::SimpleMenuModel,
 
   void RegisterLogMenuMetricsCallback(LogMenuMetricsCallback callback);
 
+  BRAVE_RECENT_TABS_SUB_MENU_MODEL_H_
  private:
   struct TabNavigationItem;
   using TabNavigationItems = std::map<int, TabNavigationItem>;
