diff --git a/chrome/browser/android/compositor/layer/tab_layer.h b/chrome/browser/android/compositor/layer/tab_layer.h
index 9f158c5906d6b6a9453c374ce0ce6e183cf7b7d8..f6f080825cb4d4e8107cb6ff80849dc66c28159e 100644
--- a/chrome/browser/android/compositor/layer/tab_layer.h
+++ b/chrome/browser/android/compositor/layer/tab_layer.h
@@ -16,6 +16,7 @@ namespace cc {
 class Layer;
 class NinePatchLayer;
 class SolidColorLayer;
+class UIResourceLayer;
 }
 
 namespace gfx {
@@ -32,10 +33,15 @@ class ContentLayer;
 class TabContentManager;
 class ToolbarLayer;
 
+class DecorationTitle;
+class LayerTitleCache;
+
 // Sub layer tree representation of a tab.  A TabLayer is not tied to
 // specific tab. To specialize it call CustomizeForId() and SetProperties().
 class TabLayer : public Layer {
  public:
+  void SetTitle(DecorationTitle* title);
+  void InitStack(LayerTitleCache* layer_title_cache);
   static scoped_refptr<TabLayer> Create(bool incognito,
                                         ui::ResourceManager* resource_manager,
                                         TabContentManager* tab_content_manager);
@@ -86,6 +92,30 @@ class TabLayer : public Layer {
                      float side_border_scale,
                      bool inset_border);
 
+  void SetStackProperties(int id,
+                          int border_resource_id,
+                          float x,
+                          float y,
+                          float width,
+                          float alpha,
+                          float border_alpha,
+                          float border_scale,
+                          float content_width,
+                          float content_height,
+                          int default_theme_color,
+                          bool inset_border,
+                          int close_button_resource_id,
+                          bool close_button_on_right,
+                          float pivot_x,
+                          float pivot_y,
+                          float rotation_x,
+                          float rotation_y,
+                          float close_alpha,
+                          float close_btn_width,
+                          float close_btn_asset_size,
+                          int close_button_color,
+                          bool show_tab_title);
+
   bool is_incognito() const { return incognito_; }
 
   scoped_refptr<cc::Layer> layer() override;
@@ -137,6 +167,10 @@ class TabLayer : public Layer {
 
   scoped_refptr<cc::NinePatchLayer> shadow_;
   float brightness_;
+
+  scoped_refptr<cc::Layer> title_;
+  scoped_refptr<cc::UIResourceLayer> close_button_;
+  raw_ptr<LayerTitleCache> layer_title_cache_;
 };
 
 }  //  namespace android
