diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 45db02025a3bedca7e354e3d38b54c2471fd7d1a..56665b6ac645ff50dd73df9bf7365590686d4b8d 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4599,6 +4599,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
