diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 1f1de831a4bab5a0a0f0b3c5ed706ef5332a91e3..31eb974d059c9f6c1cf6f50c44cf6c1fc0559b41 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4008,6 +4008,7 @@ const flags_ui::FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
