diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
index 681fe10f2826088abc2e8ebdc388ff6f4314cbcf..0e7d3e8897f6a44a3117fa91848831dfef064647 100644
--- a/chrome/browser/ui/browser.cc
+++ b/chrome/browser/ui/browser.cc
@@ -528,6 +528,7 @@ Browser* Browser::Create(const CreateParams& params) {
   // not possible, e.g. using the wrong profile or during shutdown. The caller
   // should handle this; see e.g. crbug.com/1141608 and crbug.com/1261628.
   CHECK_EQ(CreationStatus::kOk, GetCreationStatusForProfile(params.profile));
+  BRAVE_BROWSER_CREATE
   return new Browser(params);
 }
 
@@ -538,7 +539,7 @@ Browser::Browser(const CreateParams& params)
       window_(nullptr),
       tab_strip_model_delegate_(
           std::make_unique<chrome::BrowserTabStripModelDelegate>(this)),
-      tab_strip_model_(std::make_unique<TabStripModel>(
+      tab_strip_model_(std::make_unique<BraveTabStripModel>(
           tab_strip_model_delegate_.get(),
           params.profile,
           params.are_tab_groups_enabled ? TabGroupModelFactory::GetInstance()
