diff --git a/chrome/browser/extensions/extension_service.h b/chrome/browser/extensions/extension_service.h
index 10f32d09b4b789b65b120c11d716d21a352aa955..d3e037ff61c32e92e697ffb260fbd865d557fc00 100644
--- a/chrome/browser/extensions/extension_service.h
+++ b/chrome/browser/extensions/extension_service.h
@@ -459,6 +459,7 @@ class ExtensionService : public ExtensionServiceInterface,
   static base::AutoReset<bool> DisableExternalUpdatesForTesting();
 
  private:
+  friend class BraveExtensionService;
   // Loads extensions specified via a command line flag/switch.
   void LoadExtensionsFromCommandLineFlag(const char* switch_name);
 #if BUILDFLAG(IS_CHROMEOS_ASH)
