diff --git a/chrome/common/extensions/BUILD.gn b/chrome/common/extensions/BUILD.gn
index ee65f84730c3e3de840157dcf30ebef4dfed71b5..62350959ae9fcecd4ce31b159aba07625f209748 100644
--- a/chrome/common/extensions/BUILD.gn
+++ b/chrome/common/extensions/BUILD.gn
@@ -120,6 +120,7 @@ source_set("extensions") {
 
     sources += [ "api/file_system_provider_capabilities/file_system_provider_capabilities_handler.cc" ]
   }
+  import("//brave/common/extensions/sources.gni") public += brave_chrome_common_extensions_public sources += brave_chrome_common_extensions_sources deps += brave_chrome_common_extensions_deps public_deps += brave_chrome_common_extensions_public_deps
 }
 
 # TODO(zturner): Enable this on Windows. See
