diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
index 9fa3d4fc9914c50b5c71045d3b1097be1c1a05dd..586c64ed312f5c7e71cddb3750c1f6c125ce94e9 100644
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -524,4 +524,5 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents) {
 
   // This is common code for all of us. PLEASE DO YOUR PART to keep it tidy and
   // organized.
+  BRAVE_TAB_HELPERS
 }
