diff --git a/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java b/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
index fb55da3748f5fbe6f1e2768058fadcedae6a0a9d..be1c3cfe16edc87430a3a3a8cb9691686b71526b 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/SwipeRefreshHandler.java
@@ -140,7 +140,7 @@ public class SwipeRefreshHandler extends TabWebContentsUserData
      *
      * @param tab The Tab where the swipe occurs.
      */
-    private SwipeRefreshHandler(Tab tab) {
+    public SwipeRefreshHandler(Tab tab) {
         super(tab);
         mTab = tab;
         mTabObserver =
