diff --git a/chrome/installer/util/shell_util.cc b/chrome/installer/util/shell_util.cc
index a542839ee48b10e323a2b20920f661ba08272b8b..95ddff40bf5de8273219ce43eeee371c89e07549 100644
--- a/chrome/installer/util/shell_util.cc
+++ b/chrome/installer/util/shell_util.cc
@@ -369,6 +369,7 @@ void GetChromeProgIdEntries(
   app_info.delegate_clsid = install_static::GetLegacyCommandExecuteImplClsid();
 
   GetProgIdEntries(app_info, entries);
+  BRAVE_GET_CHROME_PROG_ID_ENTRIES
 
   if (!app_info.delegate_clsid.empty()) {
     auto delegate_execute_entries =
@@ -464,6 +465,7 @@ void GetShellIntegrationEntries(
 
   const std::wstring html_prog_id(GetBrowserProgId(suffix));
   for (int i = 0; ShellUtil::kPotentialFileAssociations[i] != nullptr; i++) {
+    BRAVE_GET_SHELL_INTEGRATION_ENTRIES
     entries->push_back(std::make_unique<RegistryEntry>(
         capabilities + L"\\FileAssociations",
         ShellUtil::kPotentialFileAssociations[i], html_prog_id));
@@ -488,6 +490,7 @@ void GetAppExtRegistrationEntries(
   std::wstring key_name =
       base::StrCat({ShellUtil::kRegClasses, kFilePathSeparator, ext,
                     kFilePathSeparator, ShellUtil::kRegOpenWithProgids});
+  BRAVE_GET_APP_EXT_REGISTRATION_ENTRIES
   entries->push_back(
       std::make_unique<RegistryEntry>(key_name, prog_id, std::wstring()));
 }
