diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index e8251fb256adc72d75e4d9af0d8e4eacace4675a..6ed911d11ee33cf05e366d585e0cc12a5cc0cda1 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4909,6 +4909,7 @@ const FeatureEntry::FeatureVariation kMobilePromoOnDesktopVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
