diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index a09559453d8688cad38fc74d3cdeab31e3dcc124..a1306bc8226e116e71a126d74cb8b0dd85926e4f 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -124,4 +124,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
