diff --git a/components/content_settings/renderer/content_settings_agent_impl.h b/components/content_settings/renderer/content_settings_agent_impl.h
index 28f178bfe8a0a0811b422b01698aa20479b007d9..92128d7a1b4ad425322cbfec031626b010e5f958 100644
--- a/components/content_settings/renderer/content_settings_agent_impl.h
+++ b/components/content_settings/renderer/content_settings_agent_impl.h
@@ -107,6 +107,7 @@ class ContentSettingsAgentImpl
   void SetRendererContentSettingRulesForTest(
       const RendererContentSettingRules& rules);
 
+  BRAVE_CONTENT_SETTINGS_AGENT_IMPL_H_
  protected:
   // Allow this to be overridden by tests.
   virtual void BindContentSettingsManager(
