diff --git a/components/omnibox/browser/autocomplete_controller.cc b/components/omnibox/browser/autocomplete_controller.cc
index c1633b07be62e1fe3549da6c4822dcf41ab01c7d..6cd0cfb99a6920570519d7837b79e93cfa3d96ee 100644
--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -1137,6 +1137,7 @@ void AutocompleteController::InitializeAsyncProviders(int provider_types) {
   if (provider_types & AutocompleteProvider::TYPE_SEARCH) {
     search_provider_ = new SearchProvider(provider_client_.get(), this);
     providers_.push_back(search_provider_.get());
+      BRAVE_AUTOCOMPLETE_CONTROLLER_AUTOCOMPLETE_CONTROLLER
   }
   // Providers run in the order they're added.  Add `HistoryURLProvider` after
   // `SearchProvider` because:
@@ -1342,6 +1343,7 @@ void AutocompleteController::UpdateResult(UpdateType update_type) {
     StartStopTimer();
   }
 
+  BRAVE_AUTOCOMPLETE_CONTROLLER_UPDATE_RESULT
   PostProcessMatches();
 
   bool default_match_changed = CheckWhetherDefaultMatchChanged(
