diff --git a/components/infobars/core/infobar_delegate.h b/components/infobars/core/infobar_delegate.h
index 360ea23f730a5317c4b76b2238f570fd7d05e2ac..54564bf5ec482c1da080d57e289c303670354e3b 100644
--- a/components/infobars/core/infobar_delegate.h
+++ b/components/infobars/core/infobar_delegate.h
@@ -176,6 +176,7 @@ class InfoBarDelegate {
     AUTOFILL_OFFER_NOTIFICATION_INFOBAR_DELEGATE = 105,
     AUTOFILL_ADDRESS_PROFILE_INFOBAR_DELEGATE_IOS = 106,
     ADD_TO_READING_LIST_IOS = 107,
+    BRAVE_INFOBAR_DELEGATE_IDENTIFIERS
   };
 
   // Describes navigation events, used to decide whether infobars should be
