diff --git a/components/omnibox/browser/BUILD.gn b/components/omnibox/browser/BUILD.gn
index d5c70b7c3ff17db6b45ab835beece607d5739a9c..97e5dba96da4f74e04587e91d219b7a14d8f1e00 100644
--- a/components/omnibox/browser/BUILD.gn
+++ b/components/omnibox/browser/BUILD.gn
@@ -363,6 +363,7 @@ static_library("browser") {
     "//ui/base",
     "//ui/gfx",
   ]
+  sources += brave_components_omnibox_browser_sources deps += brave_components_omnibox_browser_deps
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
 
