diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 04127f868a7de3316213fb69123a150910214d0d..47e40f9db8eb9ce271880bf975eae20a4352d91c 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -183,6 +183,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
