diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 4d12e6ca4faae217c550a7b73bf1fd522c1e6a5a..68064210a70a042abd32628dc8876ca1305c1dc5 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1231,6 +1231,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
