diff --git a/components/history/core/browser/history_types.h b/components/history/core/browser/history_types.h
index e6a993ab87d10d7828cf240975bd351df8870bf3..b7fa9f7edc4f87ac8c9c5f8adeec2bc6730c90ae 100644
--- a/components/history/core/browser/history_types.h
+++ b/components/history/core/browser/history_types.h
@@ -53,6 +53,8 @@ enum VisitSource {
   SOURCE_FIREFOX_IMPORTED = 3,
   SOURCE_IE_IMPORTED = 4,
   SOURCE_SAFARI_IMPORTED = 5,
+  SOURCE_CHROME_IMPORTED = 6,
+  SOURCE_BRAVE_IMPORTED = 7,
 };
 
 // Corresponds to the "id" column of the "visits" SQL table.
