diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index d18eb0a3c45e9b2effd6bff936487f786856467b..6b9d1fa0da5dfaf03bfe575b19f66f83339cda09 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -125,4 +125,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
