diff --git a/components/sync/service/BUILD.gn b/components/sync/service/BUILD.gn
index 1e308197c4ff7c70009107ce6c6916a79d7a6e42..c2df9fb4abee0d97eb510f6690247a7a2f6f20b2 100644
--- a/components/sync/service/BUILD.gn
+++ b/components/sync/service/BUILD.gn
@@ -126,4 +126,5 @@ static_library("service") {
   }
 
   configs += [ "//build/config/compiler:wexit_time_destructors" ]
+  sources += brave_components_sync_driver_sources deps += brave_components_sync_driver_deps
 }
