diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 3f25190aa843df7e5056ec80b7e4b435ac5124b5..70243b2c90ca7f3cabda2f966b07351d2eb435b5 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -142,6 +142,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
