diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 672f96957d850d4898c7716d9eb70ac85ae3af17..2e90e565f13b33b29a4dfae9b28e1262f1cc7c41 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -257,6 +257,7 @@ class SyncServiceImpl : public SyncService,
   void ReportDataTypeErrorForTest(ModelType type);
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
