diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 5e30a20554e63a5a98a91dd13b9af508dae94b49..0e3cb84455d62d07b83353ac531516ffa4dcbb6f 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -181,6 +181,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
