diff --git a/components/browser_ui/settings/android/BUILD.gn b/components/browser_ui/settings/android/BUILD.gn
index 7eeb961d2bfcc5a064e62d555b138ccce4564f2a..b943ceb0180408531b57305ca9822e9d4f971967 100644
--- a/components/browser_ui/settings/android/BUILD.gn
+++ b/components/browser_ui/settings/android/BUILD.gn
@@ -57,6 +57,7 @@ android_library("java") {
     "//ui/android:ui_no_recycler_view_java",
   ]
   resources_package = "org.chromium.components.browser_ui.settings"
+  import("//brave/components/browser_ui/settings/android/sources.gni") sources += brave_components_browser_ui_settings_android_java_sources
 }
 
 android_resources("java_resources") {
