diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index feaa2899577655ec68da4893491df1b19b43d6c1..707d041f89c72a3dc83c426926d1b69ce06d8e21 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -150,6 +150,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
 #endif
     case RequestType::kIdentityProvider:
       return RequestTypeForUma::PERMISSION_IDENTITY_PROVIDER;
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
@@ -175,6 +176,7 @@ RequestTypeForUma GetUmaValueForRequests(
     return RequestTypeForUma::MULTIPLE_KEYBOARD_AND_POINTER_LOCK;
   }
 #endif
+  BRAVE_GET_UMA_VALUE_FOR_REQUESTS
   return RequestTypeForUma::UNKNOWN;
 }
 
