diff --git a/components/page_info/page_info.cc b/components/page_info/page_info.cc
index 6100bfa721585e738f82fc7e47056357302117da..f787c0b92d25623961c2266abc4b9c420e6f7a69 100644
--- a/components/page_info/page_info.cc
+++ b/components/page_info/page_info.cc
@@ -1238,6 +1238,7 @@ void PageInfo::PopulatePermissionInfo(PermissionInfo& permission_info,
 // applies to permissions listed in |kPermissionType|.
 bool PageInfo::ShouldShowPermission(
     const PageInfo::PermissionInfo& info) const {
+  BRAVE_PAGE_INFO_SHOULD_SHOW_PERMISSION
   // Note |ContentSettingsType::ADS| will show up regardless of its default
   // value when it has been activated on the current origin.
   if (info.type == ContentSettingsType::ADS) {
