diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index 02d61726d50203c69039b6d6b5075251cba44584..97bc604c6a3a2fc600cad26715eb48aaedd6fd11 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -219,6 +219,7 @@ json_to_struct("prepopulated_engines") {
   schema_file = "prepopulated_engines_schema.json"
   namespace = "TemplateURLPrepopulateData"
   deps = [ ":search_engine_type" ]
+  additional_sources = brave_components_search_engines_sources
 }
 
 if (is_android) {
