diff --git a/components/permissions/android/BUILD.gn b/components/permissions/android/BUILD.gn
index d9a562ea02f4a78c5ea29d4a23ac503e6ff1e291..e46f91aa272047a7286d4031ba354828b04f6830 100644
--- a/components/permissions/android/BUILD.gn
+++ b/components/permissions/android/BUILD.gn
@@ -163,6 +163,7 @@ android_library("java") {
     "//third_party/jni_zero:jni_zero_java",
     "//ui/android:ui_java",
   ]
+  sources += brave_components_permissions_java_sources deps += brave_components_permissions_java_deps
   resources_package = "org.chromium.components.permissions"
 }
 
