diff --git a/components/download/internal/common/download_item_impl.cc b/components/download/internal/common/download_item_impl.cc
index b744024bc44c65ad4b2b959157d40aa202d08132..7a9585fe6ef5453ea308f251c573b61ede7a7ca9 100644
--- a/components/download/internal/common/download_item_impl.cc
+++ b/components/download/internal/common/download_item_impl.cc
@@ -2020,8 +2020,8 @@ void DownloadItemImpl::OnDownloadCompleting() {
                      base::Unretained(download_file_.get()),
                      GetTargetFilePath(),
                      delegate_->GetApplicationClientIdForFileScanning(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetURL(),
-                     delegate_->IsOffTheRecord() ? GURL() : GetReferrerUrl(),
+                     GURL(),  // Never leak download URLs in metadata.
+                     GURL(),  // See brave-browser#2766.
                      std::move(quarantine), std::move(rename_callback)));
 }
 
