diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index 1c07025cee6998682e22532c0a67aadc2bcd46d6..51ee933ed5b5f948b34c4b185baaca674bcd7409 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -42,6 +42,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
