diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 3658c1d779713f2b8a3e977629658dc06f88c640..bd2f5d0fa3ea59597c7c414f04ba1b0f5a95c8db 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3605,6 +3605,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
