diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 4d6b73378e1928071b2012260597639f4abc17c8..832671262be89e9c94e5d369d044917f7abd16c2 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3679,6 +3679,7 @@ source_set("browser") {
   if (is_chromeos) {
     deps += [ "//ash/constants" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
