diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index daf25e569bcc8ae20cfb222bd5ccc21255431de0..d542974b5bcff4452cff6a232d92db363c4f8afd 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -908,6 +908,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
