diff --git a/content/common/frame.mojom b/content/common/frame.mojom
index 28631bef3c6ee2b8bd590503ead3b7d62c626c60..6926e8c2396251ae672a4c829db486b4143d30e7 100644
--- a/content/common/frame.mojom
+++ b/content/common/frame.mojom
@@ -564,6 +564,7 @@ struct CreateNewWindowParams {
   // Whether the opener will be suppressed in the new window, in which case
   // scripting the new window is not allowed.
   bool opener_suppressed;
+  bool no_javascript_access;
 
   // Whether new window is created for a form submission.
   bool is_form_submission;
