diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 8b94386ec6e26a08fc8625ed8fc98a559c276692..34462405d516b3db1a645ac366ec28913e61d6ad 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2291,6 +2291,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
