diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index c40dc2a618e43a920a13d3e776447ce6a68b0c0a..95fa0c8ef50da44b42a093db7fcfdf4b91efc49b 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2292,6 +2292,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
