diff --git a/content/renderer/render_thread_impl.cc b/content/renderer/render_thread_impl.cc
index 1ddaf4ec04b813bec53fef76ba9c404e66f32996..64c679f99e0963825330fda2f2c8037c4a526b5c 100644
--- a/content/renderer/render_thread_impl.cc
+++ b/content/renderer/render_thread_impl.cc
@@ -900,6 +900,7 @@ void RenderThreadImpl::InitializeRenderer(
 }
 
 void RenderThreadImpl::RegisterSchemes() {
+  REGISTER_BRAVE_SCHEMES_DISPLAY_ISOLATED_AND_NO_JS
   // chrome:
   WebString chrome_scheme(WebString::FromASCII(kChromeUIScheme));
   WebSecurityPolicy::RegisterURLSchemeAsDisplayIsolated(chrome_scheme);
