diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 32ec2d854847d28b8c9ad665a1b8a50a6bad3100..3e752835e1f614b8caab4188a779bd5172da99e3 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3637,6 +3637,7 @@ source_set("browser") {
       "profiling_utils.cc",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
