diff --git a/extensions/browser/BUILD.gn b/extensions/browser/BUILD.gn
index 72dd41b6d79ceaa7602d1ec0d3ac587da49fe548..5c1ca83b1248a68a7b38a9c8d2f462b635fcb628 100644
--- a/extensions/browser/BUILD.gn
+++ b/extensions/browser/BUILD.gn
@@ -412,6 +412,7 @@ jumbo_source_set("browser_sources") {
     "//third_party/blink/public/mojom/frame",
     "//ui/display",
   ]
+  if (brave_chromium_build) { deps += [ "//brave/extensions/browser"] }
 
   public_deps = [
     "//content/public/common",
