diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index 18d48034118c9bd7f48060973ba39dc1c216c963..b7f8f69f30a55bccbc231b1c82dffea9acbba4f1 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1342,6 +1342,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BRAVE_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
