diff --git a/net/dns/host_resolver_manager_job.cc b/net/dns/host_resolver_manager_job.cc
index 4e361df8798fa27d93c76497e486da33d21e49b3..e2003a5c0ea2d5290bfd1f11e9b4be46e7db056b 100644
--- a/net/dns/host_resolver_manager_job.cc
+++ b/net/dns/host_resolver_manager_job.cc
@@ -1051,6 +1051,7 @@ void HostResolverManager::Job::CompleteRequests(
   }
 
   RecordJobHistograms(results, task_type);
+  BRAVE_RECORD_DNS_COUNTS
 
   // Complete all of the requests that were attached to the job and
   // detach them.
