diff --git a/net/proxy_resolution/configured_proxy_resolution_service.cc b/net/proxy_resolution/configured_proxy_resolution_service.cc
index de90d3583684df0121123e59f0671d9edaaf593a..dde504074ff0d8159b69106bfd7b7a5608618b00 100644
--- a/net/proxy_resolution/configured_proxy_resolution_service.cc
+++ b/net/proxy_resolution/configured_proxy_resolution_service.cc
@@ -1025,6 +1025,7 @@ int ConfiguredProxyResolutionService::TryToCompleteSynchronously(
   config_->value().proxy_rules().Apply(url, result);
   result->set_traffic_annotation(
       MutableNetworkTrafficAnnotationTag(config_->traffic_annotation()));
+  BRAVE_TRY_TO_COMPLETE_SYNCHRONOUSLY
 
   return OK;
 }
