diff --git a/net/dns/host_resolver_manager_job.cc b/net/dns/host_resolver_manager_job.cc
index 66fa3ddd09a444380eb5bc95e2bc91d83ba4b983..f9b6e43df9900dc8039e342a4423bc08d204f5e1 100644
--- a/net/dns/host_resolver_manager_job.cc
+++ b/net/dns/host_resolver_manager_job.cc
@@ -1069,6 +1069,7 @@ void HostResolverManager::Job::CompleteRequests(
   }
 
   RecordJobHistograms(results, task_type);
+  BRAVE_RECORD_DNS_COUNTS
 
   // Complete all of the requests that were attached to the job and
   // detach them.
