diff --git a/net/proxy_resolution/configured_proxy_resolution_service.cc b/net/proxy_resolution/configured_proxy_resolution_service.cc
index 7427f31b5470cda1fb0a4ca2bcd77db3acbe4bce..0184f59bb7def6b52e18f5667c16fd1596f8c49c 100644
--- a/net/proxy_resolution/configured_proxy_resolution_service.cc
+++ b/net/proxy_resolution/configured_proxy_resolution_service.cc
@@ -977,6 +977,7 @@ int ConfiguredProxyResolutionService::TryToCompleteSynchronously(
   config_->value().proxy_rules().Apply(url, result);
   result->set_traffic_annotation(
       MutableNetworkTrafficAnnotationTag(config_->traffic_annotation()));
+  BRAVE_TRY_TO_COMPLETE_SYNCHRONOUSLY
 
   return OK;
 }
