diff --git a/PRESUBMIT.py b/PRESUBMIT.py
index 52b942450ce4ab1695c57385c165dc9aebfa676e..05540b723bc352161cd80562c2b8ee90635081f1 100644
--- a/PRESUBMIT.py
+++ b/PRESUBMIT.py
@@ -6085,7 +6085,6 @@ def CheckPythonShebang(input_api, output_api):
                 file))
     return result
 
-
 def CheckBatchAnnotation(input_api, output_api):
     """Checks that tests have either @Batch or @DoNotBatch annotation. If this
     is not an instrumentation test, disregard."""
@@ -6148,3 +6147,5 @@ Robolectric tests do not need a @Batch or @DoNotBatch annotations.
 """, extra_annotation_errors))
 
     return results
+
+if not globals().get('__name__'): from lib.import_inline import inline_module; inline_module('chromium_presubmit_overrides', globals(), locals())
