diff --git a/services/network/cookie_manager.cc b/services/network/cookie_manager.cc
index 6bd53dbea4331a0f002d0a836d6de128abe4de80..9ea9bb19022a2bd960f88bb68f17eabf2efc5da1 100644
--- a/services/network/cookie_manager.cc
+++ b/services/network/cookie_manager.cc
@@ -319,6 +319,7 @@ CookieDeletionInfo DeletionFilterToInfo(mojom::CookieDeletionFilterPtr filter) {
   delete_info.name = std::move(filter->cookie_name);
   delete_info.url = std::move(filter->url);
   delete_info.host = std::move(filter->host_name);
+  BRAVE_DELETIONFILTERTOINFO
 
   switch (filter->session_control) {
     case mojom::CookieDeletionSessionControl::IGNORE_CONTROL:
