diff --git a/sql/database.h b/sql/database.h
index 494c52135bf2d09de2447c4a379c69a78442b085..86bfff44fac5a0ea7e0f94c9dae71845114459f2 100644
--- a/sql/database.h
+++ b/sql/database.h
@@ -297,7 +297,7 @@ class COMPONENT_EXPORT(SQL) Database {
     // Purposely not explicit to avoid requiring callers to wrap their tag
     // string.
     consteval Tag(const char* tag_value) : value(tag_value) {
-      if (!sql_metrics::IsValidDatabaseTag(tag_value)) {
+      if (false && !sql_metrics::IsValidDatabaseTag(tag_value)) {
         // This will never actually invoke what's under NOTREACHED(), but
         // NOTREACHED() is invalid in a consteval context so compilation will
         // fail iff the string is invalid.
