diff --git a/third_party/blink/common/permissions/permission_utils.cc b/third_party/blink/common/permissions/permission_utils.cc
index 82b16083339538807e7466e9b41214f8663b50ec..2cf9c8e8a456987facf8a54bad88dd1e57f37f00 100644
--- a/third_party/blink/common/permissions/permission_utils.cc
+++ b/third_party/blink/common/permissions/permission_utils.cc
@@ -28,6 +28,7 @@ mojom::PermissionStatus ToPermissionStatus(const std::string& status) {
 
 std::string GetPermissionString(PermissionType permission) {
   switch (permission) {
+    PERMISSION_UTIL_GET_PERMISSION_STRING
     case PermissionType::GEOLOCATION:
       return "Geolocation";
     case PermissionType::NOTIFICATIONS:
@@ -130,6 +131,7 @@ PermissionTypeToPermissionsPolicyFeature(PermissionType permission) {
     case PermissionType::DISPLAY_CAPTURE:
       return mojom::PermissionsPolicyFeature::kDisplayCapture;
 
+    PERMISSION_UTIL_PERMISSION_TYPE_TO_PERMISSIONS_POLICY_FEATURE
     case PermissionType::PERIODIC_BACKGROUND_SYNC:
     case PermissionType::DURABLE_STORAGE:
     case PermissionType::BACKGROUND_SYNC:
