diff --git a/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py b/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
index 6635ff4b5a72cbba0c238ec75540be80b0355c06..4d993a0578c837066e994899cde0fa32285e2c11 100644
--- a/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
+++ b/third_party/blink/renderer/bindings/scripts/bind_gen/interface.py
@@ -7583,3 +7583,4 @@ def generate_interfaces(task_queue):
                          "InitIDLInterfacesForTesting",
                          "init_idl_interfaces_for_testing",
                          for_testing=True)
+from brave_chromium_utils import inline_chromium_src_override; inline_chromium_src_override(globals(), locals())
\ No newline at end of file
