diff --git a/third_party/android_deps/BUILD.gn b/third_party/android_deps/BUILD.gn
index d2d18451377581936f91eef128af037c3de0bee5..bd1ba9ca4d876c04719aaced3c74a98f2899e594 100644
--- a/third_party/android_deps/BUILD.gn
+++ b/third_party/android_deps/BUILD.gn
@@ -467,6 +467,7 @@ java_prebuilt("com_google_auto_value_auto_value_annotations_java") {
     ":*",
     "//third_party/androidx:*",
   ]
+  visibility += [ "//brave/third_party/android_deps:com_google_android_datatransport_transport_runtime_java__classes" ] supports_android = true
 }
 
 # This is generated, do not edit. Update BuildConfigGenerator.groovy instead.
