diff --git a/tools/json_schema_compiler/idl_schema.py b/tools/json_schema_compiler/idl_schema.py
index 0d0fdb5423a5e5bc3eaf119ccd51b6935b1033aa..46e25eda7327f7a68f1344b62fea4a2a04ad567c 100755
--- a/tools/json_schema_compiler/idl_schema.py
+++ b/tools/json_schema_compiler/idl_schema.py
@@ -585,6 +585,8 @@ class IDLSchema(object):
           documentation_options['namespace'] = node.value
         elif node.name == 'documented_in':
           documentation_options['documented_in'] = node.value
+        elif node.name == 'none_as_absent_optional':
+          compiler_options['none_as_absent_optional'] = True
         else:
           continue
       else:
