diff --git a/ui/native_theme/native_theme_win.cc b/ui/native_theme/native_theme_win.cc
index 5df8ed77b0e665ee82c1b3d67f31e722f357fba2..aba8bebcdc66b318ce8412560b3d698c0a1d8af3 100644
--- a/ui/native_theme/native_theme_win.cc
+++ b/ui/native_theme/native_theme_win.cc
@@ -1626,6 +1626,7 @@ void NativeThemeWin::UpdateDarkModeStatus() {
                                     &apps_use_light_theme);
     dark_mode_enabled = (apps_use_light_theme == 0);
   }
+  BRAVE_NATIVETHEMEWIN_UPDATEDARKMODESTATUS
   set_use_dark_colors(dark_mode_enabled);
   set_preferred_color_scheme(CalculatePreferredColorScheme());
   NotifyOnNativeThemeUpdated();
