diff --git a/ui/views/controls/menu/menu_separator.cc b/ui/views/controls/menu/menu_separator.cc
index 6a895c94473f854b0cc3797338cf60005bb6bc9c..c3f0dced7984ac8336fbb939e75e66b24c58fc50 100644
--- a/ui/views/controls/menu/menu_separator.cc
+++ b/ui/views/controls/menu/menu_separator.cc
@@ -47,7 +47,7 @@ void MenuSeparator::OnPaint(gfx::Canvas* canvas) {
     paint_rect.Inset(
         gfx::Insets::TLBR(0, menu_config.padded_separator_left_margin, 0,
                           menu_config.padded_separator_right_margin));
-  } else {
+  } BRAVE_MENU_SEPARATOR_ON_PAINT else {
     paint_rect.Inset(gfx::Insets::TLBR(0, menu_config.separator_left_margin, 0,
                                        menu_config.separator_right_margin));
   }
