diff --git a/ui/webui/resources/tools/bundle_js.gni b/ui/webui/resources/tools/bundle_js.gni
index a6b7d71b983b7cba19626af2787810f764c0b6dc..6be89d32d83aaba74b5644755739461535a3ae05 100644
--- a/ui/webui/resources/tools/bundle_js.gni
+++ b/ui/webui/resources/tools/bundle_js.gni
@@ -109,6 +109,7 @@ template("bundle_js") {
     external_paths += [ "//resources/polymer/v3_0/|$polymer_path" ]
 
     if (!is_chrome_untrusted) {
+      brave_resources_path = rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/", root_build_dir) external_paths += [ "chrome://resources/brave/|$brave_resources_path", "//resources/brave/|$brave_resources_path" ]
       external_paths += [ "chrome://resources/|$resources_path" ]
     }
     external_paths += [ "//resources/|$resources_path" ]
