diff --git a/BUILD.gn b/BUILD.gn
index 96680594bdd4ac4607e8ce848b40993eb40b3990..1f5531ad1b2a07776c667cf9c89fa481b38e8eda 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -827,6 +827,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3183,6 +3184,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
