diff --git a/BUILD.gn b/BUILD.gn
index 53ad9928d641f4afa9cc6bcaa790a9d9313416c5..6fa3cc50c5ea661911eb4837c18a78fcd84b99c4 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -763,6 +763,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3184,6 +3185,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
