diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 313336f56c14cdf045120192ede042a9701c5f63..f62527c2a552b463e36fab70f539cbe494fed55e 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3225,6 +3225,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
