diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 92de21a8705a43d312106e6b6f8d4c2db42f6d13..a9cf1984d31490203dcd05d70dcb85a7c701490f 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -2989,6 +2989,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
