diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 674638fe0594233adee38c77ff8bae2ba216250f..ffe754c035529fa2047af42dd11f0fea2f2bfdaf 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3225,6 +3225,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
